src/pages/filter/ui/modal-edit/components/modal-inheritance-mode/modal-inheritance-mode.store.ts

```diff
@@ -0,0 +1,116 @@
+import { Dispatch, SetStateAction } from 'react'
+import { makeAutoObservable } from 'mobx'
+
+import { ActionType } from '@declarations'
+import dtreeStore from '@store/dtree'
+import modalEditStore from '@pages/filter/ui/modal-edit/modal-edit.store'
+import { addAttributeToStep } from '@utils/addAttributeToStep'
+import { changeFunctionalStep } from '@utils/changeAttribute/changeFunctionalStep'
+
+interface ISetProblemGroupsProps {
+  checked: boolean
+  value: string
+  groupName: string
+  setSelectedProblemGroups: Dispatch<SetStateAction<string[]>>
+}
+class ModalInheritanceModeStore {
```

Comment:
I think that this class is not a store. It does not have any data or getters. May be it will be better to create simple utility functions?